verifier/src/report_body.rs

```diff
@@ -221,37 +222,39 @@ mod test {
 
     #[test]
     fn attributes_success() {
-        let report_body = ReportBody::from(&REPORT_BODY_SRC);
-        let verifier = AttributesVerifier::new(REPORT_BODY_SRC.attributes.into());
+        let attributes = Attributes::default()
+            .set_flags(1)
```

Comment:
Does `1` refer to a specific flag? If so, I'd recommend using a variable that names the flag being used.